rootconf/default/bin/exec_installer

```diff
@@ -284,6 +284,9 @@ $onie_disco_siaddr
 $onie_disco_serverid
 # Try TFTP server IP as HTTP
 $onie_disco_tftpsiaddr
+# Try DHCP TFTP server name (opt 66) as HTTP
+# Requires DNS
+$onie_disco_tftp
```

Comment:
This looks nice.  Thanks.

If you put "Closes: #476" in the commit message, then this issue will be automatically closed when the commit is pushed to master.

See https://help.github.com/articles/closing-issues-via-commit-messages/